widgets/HTTP-streamingsource.json

```diff
@@ -676,6 +700,30 @@
         }
       ]
     },
+    {
+      "name": "Authenticate with Grant type",
+      "condition": {
+        "expression": "authType == 'oAuth2' && oauth2GrantType == 'Client Credentials'"
+      },
+      "show": [
+        {
+          "name": "oauth2ClientAuthentication",
+          "type": "property"
+        }
+      ]
+    },
+    {
+      "name": "Show Refresh_Token when Grant type is Refresh Token",
+      "condition": {
+        "expression": "authType == 'oAuth2' && oauth2GrantType == 'Refresh Token'"
```

Comment:
I think this condition should be rephrased to also tackle condition when `oauth2GrantType` is `null` for older pipelines, we should do something like `oauth2GrantType != 'Client Credentials'`